src/main/webapp/WEB-INF/views/index.html

```diff
@@ -93,6 +93,16 @@
     <!-- GitHub Fork Ribbon -->
     <link th:href="@{/plugins/gh-fork-ribbon/gh-fork-ribbon.min.css}" rel="stylesheet"/>
 
+    <!-- Leaflet (OSM javascript library) -->
+    <link th:href="@{/plugins/leaflet/css/leaflet.css}" rel="stylesheet"/>
+
+    <!-- jsonFormatter css -->
+    <link th:href="@{/plugins/json-formatter/json-formatter.min.css}" rel="stylesheet"/>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/2.3.1/jspdf.umd.min.js"></script>

```

Comment:
For version control reasons, we should store these files locally and refer them via thymeleaf (like we do for the other scripts).